src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -15,8 +15,8 @@
 use Symfony\Component\Form\Extension\Core\DataTransformer\ArrayToPartsTransformer;
 use Symfony\Component\Form\Extension\Core\DataTransformer\DataTransformerChain;
 use Symfony\Component\Form\Extension\Core\DataTransformer\DateTimeToArrayTransformer;
+use Symfony\Component\Form\Extension\Core\DataTransformer\DateTimeToHTML5DateTimeLocalTransformer;
```

Comment:
case mismatch (see failing tests): should be `Html5` (same

Reply:
Oops! Thanks.